BCI_analysis/plot/plot_imaging.py

```diff
@@ -263,3 +264,95 @@ def plot_trialwise_closed_loop(dataDict, dayOfInterest):
     plt.show()
 
 
+def correlation_deltaActivity(dataDict, dayOfInterest):
+    ## Set Up Day of Session
+    # dayOfInterest = input('Day of interest (day 1 = -1... dont do day 1): ')
+    if int(dayOfInterest) == 0:
+        while int(dayOfInterest) <= 0: #this is so we dont get stupid values
+            print('Incorrect Day Entry, redo please')
+            dayOfInterest = input('Day of interest (Day 1 = "1") ')
+    day = int(dayOfInterest) - 1 #Since python starts at 0, we are subtracting one to account for indexing
+    print('Interested in day '+dayOfInterest)
+
+
+    #Grab Day-Specific Data
+    today_dff_allTrials = dataDict['dff_trialwise_closed_loop'][day]
+    yesterday_dff_allTrials = dataDict['dff_trialwise_closed_loop'][day-1]
+
+    if 'conditioned_neuron_idx' in dataDict.keys(): #some sessions dont have a conditioned neuron
+        todays_conditioned_neuron_index = dataDict['conditioned_neuron_idx'][0][day] #this is the 'cn' field in mat file
+        # yesterdays_conditioned_neuron_index = dataDict['conditioned_neuron_idx'][0][day-1]#this is the 'cn' field in mat file
+    else:
+        print('Some Sessions do not include conditioned neurons -- cannot use dataset')
+        quit()
```

Comment:
Im not sure what quit() does. if it quits the whole program, maybe we should use 
`return` 
instead